EIPS/eip-191.md

```diff
@@ -55,29 +54,55 @@ Using `0x19` thus makes it possible to extend the scheme by defining a version `
 |    `0x01`    | [712][eip-712] | Structured data
 |    `0x45`    | [191][eip-191] | `personal_sign` messages
 
+#### Version `0x00`
+
+```
+0x19 <0x00> <intended validator address> <data to sign>
```

Comment:
Wait, I must be missing something here, are we adding the `<0x00> <intended validator address>` to the original EIP-191?

Reply:
@xinbenlv  The version `0x00` was always there in the standard from the start, but it was not explained very well, and that's the purpose of the PR